src/main/java/com/vmware/vipclient/i18n/messages/api/opt/server/ComponentBasedOpt.java

```diff
@@ -35,9 +35,6 @@ public ComponentBasedOpt(final MessagesDTO dto) {
     public void getComponentMessages(MessageCacheItem cacheItem) {
         String url = V2URL.getComponentTranslationURL(this.dto,
                 VIPCfg.getInstance().getVipService().getHttpRequester().getBaseURL());
-        if (ConstantsKeys.LATEST.equals(this.dto.getLocale())) {
-            url = url.replace("pseudo=false", "pseudo=true");
-        }
```

Comment:
Could you explain what problem do these lines resolve before you remove them?

Reply:
Nothing, really. Here, it says that if locale is latest", force it to use pseudo=true, which means the result will have #@ around it. This does not even makes sense because if you look at lines 84-87 below, it was stripping all the #@ from the response anyway, forcing the messages to ALWAYS look like pseudo=false if locale is "latest". 

We do not need to force pseudo=false for "latest". Removing this allows the product team to apply the same pseudo configuration, whether it is true or false, to all locale bundles including "latest".